macros/MakePlots.py

```diff
@@ -93,26 +94,20 @@ def LoadInputs(self):
         self.graphs = OrderedDict()
         self.files[self.fname] = rt.TFile(os.path.join(self.inputPath,f'{self.fname}.root'))
         f_ = self.files[self.fname]
+        # remove response names, that do not exist
+        lista = [ el.GetName().split("_")[0] for el in f_.GetListOfKeys() if "response" in el.GetName()]
```

Comment:
lists?

Reply:
Well, you somehow need to check if the histograms are in the file for calculating the response, such that you not always need to adapt the list above with the responses you want to plot. So I though that is the easiest way. If you know a better one I am happy to change :)

Reply:
lists, not lista. It's just a typo :-)

Reply:
ah, no I do that on purpose :D I always call them lista (List A), but I can also change that if it bothers you?

Reply:
I guess it feels weird to us because `lista` is the italian word for `list` :)
It find it weird but I have no strong opinion